### DISCUSSION TITLE
[Build]: Menu pop-up issue

### DISCUSSION THREAD

**Discussion by lisarun** [YimMenu/YimMenu#2954] [Q&A]
### Describe the build issues

I tried to modify YimMenu, but after compilation, I tried to inject it. After injection, a menu would automatically pop up. I want to hide the menu, but I can't find the code to which it belongs. Can you tell me which cpp file this hotkey is in?
I don't want it to pop up automatically, I want to hide it, so I want to implement it in the code, but I can't find the CPP file it belongs to

### Screenshots and/or Logs

![1111](https://github.com/YimMenu/YimMenu/assets/147649364/1e49b2e6-edc2-4beb-90b6-f45b3b1bba00)

### IDE/Compiler

Visual Studio 2022

### Setup Guide

- [X] I confirm that I followed the setup guide correctly.

### YimMenu Source Code

- [X] I've confirmed this is an issue with building unmodified YimMenu.

**Comment by Yimura** [comment]
This is not a build issue, at best this is worthy of a discussion. However whatever changes you make is for you to figure out. Get familiar with the code base and find out where it is.

If you want to continue this conversation make a discussion.

**Comment by lisarun** [comment]
I searched all afternoon today and tried many times, but couldn't find it, so I came to seek help. I hope you can help me. My good big brother

**Reply by Yimura** [reply]
https://github.com/YimMenu/YimMenu/blob/master/src/views/core/view_onboarding.cpp

**Reply by lisarun** [reply]
Okay, I'll give it a try, dear brother. Thank you very much for your help

**Reply by lisarun** [reply]
Thank you, big brother. Thank you very much. My problem has been solved

**Comment by lisarun** [comment]
@Yimura   I really hope you can help me with this issue. I couldn't find the hotkey code for calling out and hiding this menu. Thank you very much, my good brother.

**Comment by lisarun** [comment]
Please help me, who will help me? Thank you, big brothers
